### DISCUSSION TITLE
Does having keyframe every 8 seconds is consider has normal?

### DISCUSSION THREAD

**Discussion by stephane-archer** [mifi/lossless-cut#1026] [Q&A] [answered]
I have a file that came from OBS that is multiple hours long, I use the default OBS settings, and when I "seek next keyframe" It moves the cursor by 8 seconds. Do you think it's the expected behavior? I didn't expect to have a keyframe only every 8 seconds.

**Comment by mifi** [comment, marked as answer]
Yes this is normal, especially for content that doesnt have much movement, typically like screencasts. If you believe it may be wrong, maybe you can try some other software and see if it shows the same 8 sec interval between keyframes
